lws_config.h.in

```diff
@@ -61,3 +61,5 @@
 
 /* use SHA1() not internal libwebsockets_SHA1 */
 #cmakedefine LWS_SHA1_USE_OPENSSL_NAME
+
+${LWS_SIZEOFPTR_CODE}
```

Comment:
Thanks for the detailed explanation. This is all good. I just hadn't been aware of the `${VAR_CODE}` stuff. This is really useful to know about :)

Your patch is fine as it is!